docker/deploy.sh

```diff
@@ -4,8 +4,6 @@ set -euo pipefail
 
 image_name=$1
 
-sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
```

Comment:
should this be `sudo apt-get update && sudo apt-get install awscli`?

Reply:
No, because it was already installed in the context before calling this script